docs/source/verdi/verdi_user_guide.rst

```diff
@@ -280,7 +280,8 @@ Setup and manage computer objects.
   *  **setup**: creates a new computer object
   *  **show**: shows the details of an installed computer
   *  **test**: tests if the current user (or a given user) can connect to the computer and if basic operations perform as expected (file copy, getting the list of jobs in the scheduler queue, ...)
-  *  **update**: change configuration of a computer. Works only if the computer node is a disconnected node in the database (has not been used yet)
+  *  **duplicate**: setup a new computer, starting from the settings of an existing one
+  *  **update**: [deprecated: use ``verdi computer duplicate`` instead] change configuration of a computer. Works only if the computer node is a disconnected node in the database (has not been used yet)
```

Comment:
Not even sure we should include deprecated functions in the user documentation.
Once the deprecation is clear from the verdi output, I think we don't need it.

(btw: in the case of verdi code, update is not just deprecated - it also does not work)